source/Camera/Maintenance/CorrectCameraTrueness/CorrectCameraTrueness.cpp

```diff
@@ -0,0 +1,119 @@
+    {
+        std::vector<Zivid::Experimental::Calibration::InfieldCorrectionInput> dataset;
+        std::cout << "Please point the camera at a Zivid in-field calibration object. " << std::endl;
+
+        constexpr char printLine[] = "------------------------------------------------------------------------";
+        while(true)
+        {
+            std::cout << printLine << std::endl;
+            if(yesNoPrompt("Capture (y) or finish (n)?"))
+            {
+                std::cout << "Capturing calibration object" << std::endl;
+                const auto detectionResult = Zivid::Experimental::Calibration::detectFeaturePoints(camera);
+                const auto input = Zivid::Experimental::Calibration::InfieldCorrectionInput{ detectionResult };
+
+                if(input.valid())
+                {
+                    dataset.emplace_back(input);
+                    std::cout << "Valid measurement at: " << input.detectionResult().centroid() << std::endl;
+                }
+                else
+                {
+                    std::cout << "****INVALID****" << std::endl;
+                    std::cout << "Feedback: " << input.statusDescription() << std::endl;
+                }
+                std::cout << printLine << std::endl;
+            }
+            else
+            {
+                std::cout << "End of capturing stage." << std::endl;
+                std::cout << printLine << std::endl;
+                break;
+            }
+            std::cout << "You have collected " << dataset.size() << " valid measurements so far." << std::endl;
+        }
+        return dataset;
+    }
+
+} // namespace
+
+int main()
+{
+    try
+    {
+        Zivid::Application zivid;
+        std::cout << "Connecting to camera" << std::endl;
+        auto camera = zivid.connectCamera();
+
+        // Gather data
+        const auto dataset = collectDataset(camera);
+
+        // Calculate in-field correction
+        std::cout << "Collected " << dataset.size() << " valid measurements." << std::endl;
+        std::cout << "Calculating new camera correction..." << std::endl;
+        const auto correction = Zivid::Experimental::Calibration::computeCameraCorrection(dataset);
+        const auto accuracyEstimate = correction.accuracyEstimate();
+        std::cout
+            << "If written to the camera, this correction can be expected to yield a dimension accuracy of "
+            << std::fixed << std::setprecision(2) << 100.0 * accuracyEstimate.dimensionAccuracy()
+            << "% or better in the range of z=[" << static_cast<int>(std::round(accuracyEstimate.zMin())) << ","
+            << static_cast<int>(std::round(accuracyEstimate.zMax()))
+            << "] across the full FOV. Accuracy close to where the correction data was collected is likely better."
+            << std::endl;
+
+        // Optionally save to camera
```

Comment:
https://github.com/zivid/zivid-cpp-samples/pull/148/files#r533510067

Reply:
Will avoid these comments. Changing to print where applicable.